src/store/modules/accounts/index.js

```diff
@@ -57,8 +57,11 @@ export default {
       return dispatch(`${getModule(active).name}/sign`, data);
     },
 
-    signTransaction({ getters: { active, getModule }, dispatch }, txBase64) {
-      return dispatch(`${getModule(active).name}/signTransaction`, txBase64);
+    signTransaction({ getters: { active, getModule }, dispatch }, { txBase64, opt }) {
```

Comment:
In scenario of "evil" sdk, sdk would be able to change signer account, and if the field modal is `false` user will not even notice.

Reply:
We set the default value for modal in the `hdWallet` file